test/unit/alignment/multiple/align_multiple_test.cpp

```diff
@@ -11,24 +11,51 @@
 #include <string_view>
 #include <vector>
 
+#include <seqan3/alignment/configuration/align_config_gap.hpp>
 #include <seqan3/alignment/multiple/align_multiple.hpp>
+#include <seqan3/alignment/pairwise/align_pairwise.hpp>
 #include <seqan3/alphabet/aminoacid/aa27.hpp>
 #include <seqan3/alphabet/gap/gapped.hpp>
+#include <seqan3/alphabet/nucleotide/dna15.hpp>
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
 #include <seqan3/range/views/char_to.hpp>
 #include <seqan3/test/expect_range_eq.hpp>
 
+#include <stdexcept>
+#include <type_traits>
+#include <utility>
+
+#include <meta/meta.hpp>
+
+#include <seqan3/alignment/configuration/all.hpp>
+#include <seqan3/alignment/pairwise/align_pairwise.hpp>
+#include <seqan3/alphabet/gap/gapped.hpp>
+#include <seqan3/alphabet/nucleotide/all.hpp>
+#include <seqan3/core/concept/tuple.hpp>
+#include <seqan3/range/views/to_char.hpp>
+#include <seqan3/range/views/to.hpp>
+#include <seqan3/std/ranges>
+
+#include <range/v3/view/generate_n.hpp>
```

Comment:
Please clean up the includes:

- add them to the already existing include groups above
- avoid `all.hpp` includes (they raise compile time and include unnecessary things)
- remove unused includes

Reply:
Sorry I forgot that before committing. This really was a mess !